build.gradle

```diff
@@ -1,72 +1,72 @@
 plugins {
-	id 'java'
-	id 'org.springframework.boot' version '2.5.3'
-	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
```

Comment:
I like this formatting. Normally it does not affect the Google style warnings, but it's in the spirit, let's keep it.